bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -24,22 +24,12 @@
         {% if not hide_nav_download_button %}
         <div class="mzp-c-navigation-download">
           {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='protocol-nav-download-firefox', button_color='mzp-t-secondary mzp-t-small', download_location='nav') }}
-          {% if LANG in ['en-US', 'de', 'fr'] %}
-            {% if LANG == 'de' %}
-              {% set fxa_button_text = 'Firefox-Konto erstellen' %}
-              {% set fxa_link_text = 'Das sind deine Vorteile' %}
-            {% elif LANG == 'fr' %}
-              {% set fxa_button_text = 'Créez un compte Firefox' %}
-              {% set fxa_link_text = 'Découvrez les avantages' %}
-            {% else %}
-              {% set fxa_button_text = 'Get a Firefox Account' %}
-              {% set fxa_link_text = 'Check out the Benefits' %}
-            {% endif %}
+          {% if LANG in ['en-US', 'en-CA', 'en-GB', 'en-ZA', 'de', 'fr'] %}
             <div class="c-navigation-fxa-cta-container">
-              <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-download" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="button" data-cta-position="secondary cta">
-                {{ fxa_button_text }}
+              <a class="c-navigation-fxa-cta mzp-c-button mzp-t-secondary mzp-t-small mzp-t-download" {{ fxa_link_fragment(entrypoint='mozilla.org-globalnav', action='signup', utm_params={'campaign': 'globalnav', 'content': 'get-firefox-account', 'medium': 'referral', 'source': 'www.mozilla.org'}) }} data-button-name="Get a Firefox Account" data-link-type="button" data-cta-position="secondary cta" data-alt-href="{{ url('firefox.accounts') }}">
+                {{ _('Get a Firefox Account') }}
```

Comment:
Not a blocker, but I wonder if there could be another string to show to people who are already signed in, something that speaks to the features/benefits of their account? Maybe something we could add in a future iteration?

Reply:
Ooh, we already have the string "Your Firefox Account" in the nav. Maybe that could work?

Reply:
Hmm... I would expect that to lead me to a place where I can manage my account, not just a marketing page. What if instead of the benefits page we link to `https://accounts.firefox.com/signin` ? Maybe not right now but that's an idea for the future.

Reply:
This is the text we already use in the nav, so seems consistent at least? My issue was more with the copy, but maybe let's leave this for now?

Reply:
Yeah, and at least it's better than "Get" when you clearly already have one. Maybe we should ask Jennifer for a UX opinion before we make the decision.

Reply:
So I've tinkered a bit and this has the same issue as hiding the benefits link; the "Get" button is visible for a second before we can get the account state and change it to "Your." We could hide everything, show no button at all until we determine state and that's how we decide whether to show "Get" or "Your" ... but then Download will be visible for a split second instead.

Maybe this will be fixed when we fix `getFxaDetails` if we no longer have to wait until the DOM finishes, but for the moment, and as long as there's some delay before we can determine signed-in/signed-out state (either for domready or uitour or something else), I don't see a way to change the button label or hide the benefits link without that little jolt of movement in the corner of your eye right after the page loads. And if this was lower down on the page I wouldn't worry about it, but right there in the navbar it really stands out. At least changing the href is invisible.

Reply:
> Maybe this will be fixed when we fix getFxaDetails if we no longer have to wait until the DOM finishes

I've submitted a PR to fix this issue, so hopefully it might help a bit? We do have pages where the entire main CTA is dependent on signed-in state and suffers from the same issue, so I don't know if this is a real show stopper or not. People who are using FF and already signed-in are in a definite minority, so it could arguably be better for only those users to see a flash of content, as opposed to making everyone wait to see the critical content?

That said, if you think it's too distracting to not be worth it, we could totally just stick to the original plan and only update the href. I think this still works OK seeing as the destination is `/accounts/`.

Reply:
I switched to visibility:hidden so it's still briefly visible, but at least once it hides it doens't make the whole page jump. I can live with that.